engines/ultima/ultima8/audio/music_process.h

```diff
@@ -37,11 +37,27 @@ class MidiPlayer;
 class MusicProcess : public Process {
 	friend class Debugger;
 
-	enum MusicStates {
-		MUSIC_NORMAL = 1,
-		MUSIC_TRANSITION = 2,
-		MUSIC_PLAY_WANTED = 3
+	enum PlaybackStates {
```

Comment:
I changed the name of this enum (and entries) to be more clear now that we also have "track" state.